src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2456,17 +2475,21 @@ out _copyLocalFiles
                     bool useNetStandard = false;
                     foreach (var reference in dependencyTable.References.Keys)
                     {
-                        if (string.Equals(SystemRuntimeAssemblyName, reference.Name, StringComparison.OrdinalIgnoreCase))
+                        if (!useSystemRuntime && string.Equals(SystemRuntimeAssemblyName, reference.Name, StringComparison.OrdinalIgnoreCase))
```

Comment:
Are these changes related? Can you separate them out?

Reply:
They are separated out into their own commit.

Reply:
ok, can you rebase and squash then if you want to preserve that commit?